code/modules/mob/living/simple_animal/hostile/bear.dm

```diff
@@ -69,6 +69,10 @@
 	maxHealth = 120
 	armored = TRUE
 
+// No more murder!
+/mob/living/simple_animal/hostile/bear/russian/hunter/ListTargets()
+	return list()
```

Comment:
Shouldn't this just not attack the people on the ship (people have russian faction)

Reply:
No, since it could potentially kill the fugitive who the russians want alive